lua/nvim-tree/renderer/components/floating-node.lua

```diff
@@ -0,0 +1,82 @@
+local M = {}
+
+local api = vim.api
+local fn = vim.fn
+
+local function hide(win)
+  if win then
+    if api.nvim_win_is_valid(win) then
+      api.nvim_win_close(win, true)
+    end
+  end
+end
+
+local popup_win = nil
+local function show()
+  hide(popup_win)
+  local line_nr = api.nvim_win_get_cursor(0)[1]
+  if line_nr == 1 then
```

Comment:
We need to call `is_root_folder_visible` to decide whether to display line 1.

Reply:
I forgeted this😄